site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -211,14 +211,12 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `gameservers.podPreserveUnknownFields`                   | Disable [field pruning][pruning] and schema validation on the Pod template for a [GameServer][gameserver] definition | `false`                |
 | `helm.installTests`                                      | Add an ability to run `helm test agones` to verify the installation                             | `8000`                 |
 
-{{% feature publishVersion="1.25.0" %}}
```

Comment:
We should have a comment for this, this section is a special case.

For this section, take the configuration parameters from the table and move them up in the table above, and then increment the `publishVersion` in this featureCode block.

Basically we always keep a small "New Configuration Features" table ongoing between releases.